go.sh

```diff
@@ -20,5 +20,36 @@
 #    files (e.g. signed public releases should match signed intel releases).
 #    "publish" rule. 
 
-echo "Nothing implemented"
+ROOT=/
+INTEL_PATH=lib/firmware/intel
+VERSION=v1.5
```

Comment:
Can you use longer variable names to minimize collision risks? Examples: FS_ROOT, INTEL_FW_PATH, SOF_VERSION,...

Reply:
I can but this code I didn't write. I just pull it into this branch. Do you want this to be like in 1.4x with minimal changes?

Reply:
> I can but this code I didn't write. I just pull it into this branch

I can't tell that from your commit message, it just says "add back" but not where from. If you used `git cherry-pick` you can use the `-x` option.